launch/board_motion_detection.launch

```diff
@@ -0,0 +1,11 @@
+<!--Listening for any movement on the TTT board region of the image -->
+<launch>
+  <include file="$(find baxter_tictactoe)/launch/usb_cam_node.launch" />
+
+  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find baxter_tictactoe)/custom_rosconsole.conf"/>
+
+  <node name="board_motion_detection" pkg="baxter_tictactoe" type="board_motion_detection" respawn="false" output="screen">
+    <remap from="/in" to="/usb_cam/image_raw"/>
+  </node>
+
+</launch>
```

Comment:
what does this launch file do? Is it necessary? I put the comment to remember us in sight of #8